Tests/SpeziLocalStorageTests/LocalStorageTests.swift

```diff
@@ -12,12 +12,26 @@ import XCTSpezi
 
 
 final class LocalStorageTests: XCTestCase {
-    struct Letter: Codable, Equatable {
+    private struct Letter: Codable, Equatable {
```

Comment:
Might be a nice opportunity to move to swift testing.
But definitely doesn't have to be done within this PR, feel free to create an issue 🚀

Reply:
And thanks for the extensive new tests!

Reply:
We should add a set of issues across all repos (we can follow the template I created for the concurrency issues) that we should start adopting Swift Testing where we can 👍